src/main/common/maths.c

```diff
@@ -16,6 +16,7 @@
  */
```

Comment:
Looks like you included the changes in maths.{c,h} by mistake

Reply:
No mistake here. I need a median filter for int16_t